model/finance.php

```diff
@@ -91,7 +93,8 @@ function __construct(Uzivatel $u, int $zustatek) {
     $this->stav =
       - $cena
       + $this->platby
-      + $this->zustatek;
+      + $this->zustatek
+      + $this->stavSlevy;
```

Comment:
Tady se to nedá jednoduše přičíst, protože ten design po nás chce, aby se sleva "nedala vybrat", tj. i příklad "mám zaplatit 750, mám slevu 999 a poslal jsem si na účet 500" by měl vyplivnout "stav účtu 500". Viz proměnná $sleva výš, pravděpodobně to bude copy/paste toho samého, mělo by se to započítat i ve stejném kroku (tj. po započtení "využitých služeb" a před započtením plateb a dobrovolného vstupného).

Reply:
Doufám, že jsem to pochopil správně :smile: